src/main/java/nextstep/subway/path/application/PathService.java

```diff
@@ -23,6 +25,7 @@ public PathResponse findPath(Long source, Long target, PathType type) {
         Station sourceStation = stationService.findStationById(source);
         Station targetStation = stationService.findStationById(target);
         PathResult pathResult = subwayGraph.findPath(sourceStation, targetStation);
-        return PathResponse.of(pathResult);
+        Fare fare = new Fare(new Distance(pathResult.getTotalDistance()));
```

Comment:
Fare를 생성하기 위해서 Distance를 생성해주고 있는데 Distance 생성을  PathService에서 할 필요가 있을까요?
만약 Fare를 생성하기 위해 PathResult를 인자로 받는다면 PathService는 Distance에 의존하지 않아도 될 것 같아서요 :)
그리고 값을 꺼내서 던지는 것 보다 객체 자체와 관계를 맺는건 어떨까요?
아래와 같은 형태로 변할 수 있는데 이 부분에 대해서 고민해주시면 좋을 것 같습니다 :) 

```suggestion
        Fare fare = new Fare(pathResult);
```